tests/validation_suite/automation_framework/utilities/post_request.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import json
+import logging
+
+import automation_framework.work_order_submit.work_order_submit_utility as \
+    wo_utility
+import automation_framework.work_order_get_result. \
+    work_order_get_result_utility as wo_get_result
+import automation_framework.worker.worker_utility as worker_utility
+from automation_framework.utilities.workflow import submit_request, \
+    is_valid_params
+
+logger = logging.getLogger(__name__)
+
+
+def post_request(request_tup):
+    """
+    Function to submit JSON-RPC requests and extracts parameters of
+    APIs's requests from request_tup.
```

Comment:
What is request_tup. Please add a comment

Reply:
Done